testsuite-osgi/src/test/java/io/netty/osgitests/OsgiBundleTest.java

```diff
@@ -104,4 +133,127 @@ public void testResolvedBundles() {
         // No-op, as we just want the bundles to be resolved. Just check if we tested something
         assertFalse("At least one bundle needs to be tested", BUNDLES.isEmpty());
     }
+
+    @Test
+    public void testSimpleEcho() throws InterruptedException {
+
+        assertFalse(PlatformDependent.hasUnsafe());
```

Comment:
Wouldn’t it be good enough to only do this? This should throw otherwise

Reply:
That is a useful check - I just thought it might be a good idea to do a more comprehensive sniff test by actually firing up a Channel in OSGi. Feel free to modify!

Reply:
I would prefer to keep the test as simple as possible